src/components/Buttons.js

```diff
@@ -1,23 +1,34 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-/* eslint-disable react/destructuring-assignment */
-/* eslint-disable react/prop-types */
 /* eslint-disable react/button-has-type */
-const Button = (props) => (
-  <button
-    id={props.id}
-    className={props.className}
-    onClick={props.handleClickEvent}
-    name={props.name}
-    type={props.type}
-  >
-    {props.name}
-  </button>
-);
+import PropTypes from 'prop-types';
 
+const Button = (props) => {
+  const {
+    id, className, handleClickEvent, name, type,
+  } = props;
+  return (
+    <button
+      id={id}
+      className={className}
+      onClick={handleClickEvent}
+      name={name}
+      type={type}
```

Comment:
[OPTIONAL] You don't have to make props for type of your button. Otherwise linters may show an error for it.